src/tools/pub/pubspec.md

```diff
@@ -184,8 +184,7 @@ authors:
 - Bob Nystrom <rnystrom@google.com>
 {% endprettify %}
 
-If anyone uploads your package to the Pub site, your email address is
-public.
+In order to authenticate with the pub server your `email` **must** be a Google or Gmail email address. If anyone uploads your package to the Pub site, your email address is public.
```

Comment:
This is not accurate.

You email can be whatever. When you do `pub publish` you will be required to authenticate with a "Google Account". This can be a Gmail account, but you can use any Google account, including one registered with your own email address.

This allows us to leverage all the security and account recovery features associated with Google Accounts.

Reply:
Okay, so then the information I was given was not clear to me. I'll make the change.